go.mod

```diff
@@ -29,13 +29,14 @@ require (
 	github.com/prometheus/client_golang v1.4.1
 	github.com/prometheus/procfs v0.0.10 // indirect
 	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/viper v1.7.0 // indirect
```

Comment:
@donutloop Please clean up the go modules with `go mod tidy`

Reply:
Ok I gonna clean up the mod files

Reply:
fixed